x/exchange/keeper/market.go

```diff
@@ -28,9 +28,14 @@ func (k Keeper) CreateMarket(
 
 	marketId := k.GetNextMarketIdWithUpdate(ctx)
 	fees := k.GetFees(ctx)
+	defaultMinOrderQty := k.GetDefaultMinOrderQuantity(ctx)
+	defaultMinOrderQuote := k.GetDefaultMinOrderQuote(ctx)
+	defaultMaxOrderQty := k.GetDefaultMaxOrderQuantity(ctx)
+	defaultMaxOrderQuote := k.GetDefaultMaxOrderQuote(ctx)
 	market = types.NewMarket(
 		marketId, baseDenom, quoteDenom,
-		fees.DefaultMakerFeeRate, fees.DefaultTakerFeeRate, fees.DefaultOrderSourceFeeRatio)
+		fees.DefaultMakerFeeRate, fees.DefaultTakerFeeRate, fees.DefaultOrderSourceFeeRatio,
+		defaultMinOrderQty, defaultMinOrderQuote, defaultMaxOrderQty, defaultMaxOrderQuote)
 	k.SetMarket(ctx, market)
 	k.SetMarketByDenomsIndex(ctx, market)
 	k.SetMarketState(ctx, market.Id, types.NewMarketState(nil))
```

Comment:
In the function of `EscrowCoins` and `ReleaseCoins`, the checking to see if `coins` is all zero might be needed.